### PR TITLE
Additonal check for dataparallel, loss_lr_scheduler support

### PR DESCRIPTION
**Pull request by oguzhanbsolak**
There are two updates:

1. Additional check for normalizing dataparallel keys when loading a checkpoint. When using dataparallel, at first run a comp. scheduler is created from a dataparallel model. Then when saving the checkpoint, both model and scheduler are saved with dataparallel keys. On the second run, when loading checkpoint, _load_compression_scheduler(): function catches an exception and returns "normalize_dataparallel_keys" as True.  Then using this flag, dataparallel keys are removed from the checkpoint. The problem starts at this point because the model will be sent to dataparallel again in the main loop and will be saved as dataparallel. However, comp. scheduler keys will remain as normalized, and  _load_compression_scheduler(): will not catch an exception. Therefore, when trying to load for the third time with dataparallel missing_leys value error will be raised. To prevent this, I added an additional check that directly searches for the "module" keyword in the checkpoint state dict.

2. Loss lr scheduler support will enable the usage of an additional lr policy that schedules the second (loss) optimizer. For the faceID case after trials, I didn't have to use this. However, it may be useful for applications in the future.

### CODE REVIEW

**Comment by oguzhanbsolak** [comment]
Updated the solution for the first item with the latest commit
